keps/sig-cloud-provider/providers/20191206-cloud-provider-huaweicloud.md

```diff
@@ -0,0 +1,141 @@
+
+### Non-Goals
+
+The networking and storage support for Kubernetes are out of the scope that will be provided by other projects.
+
+## Prerequisites
+
+- Kubernetes version must be v1.17+.
+- Go version must be v1.13+.
+
+### Repository Requirements
+
+[HUAWEI CLOUD Controller Manager](https://github.com/huawei-cloudnative/cloud-provider-huaweicloud) is an implementation of
+[Kubernetes Cloud Controller Manager](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/). 
+
+### User Experience Reports
+
+`Huawei Technologies` is the founding members of CNCF, and `HUAWEI CLOUD` provides customers with stable, reliable, 
+secure, and sustainably growing cloud services. It helps large enterprises address challenges in cloud transformation 
+and enables them to take better advantages of potential business opportunities. 
+It also helps small- and medium-sized enterprises expand their business growth and rise to challenges. 
+
+The [Cloud Container Engine (CCE)](https://www.huaweicloud.com/en-us/product/cce.html) is a high-performance, 
+high-reliability service through which enterprises can manage containerized native Kubernetes applications. 
+CCE has been using this project for several releases:
+- v1.15 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.15/huawei-cce)
+- v1.13 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.13/huawei-cce)
+- v1.11 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.11/huawei-cce)
+- v1.9 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.9/huawei-cce)
+
+As well as CCE, another certified Kubernetes product which used this project is `Huawei Fusionstage`, and you can refer to 
+following conformance test reports:
+- v1.15 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.15/huawei-fusionstage)
+- v1.13 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.13/huawei-fusionstage)
+- v1.11 [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.11/huawei-fusionstage)
+- v1.9  [Conformance Test Report](https://github.com/cncf/k8s-conformance/tree/843ee84d40962baa07cab9e59a19abe7f778b6b0/v1.9/huawei-fusionstage)
+
+Other usage of this project can be seen from [GitHub issues](https://github.com/huawei-cloudnative/cloud-provider-huaweicloud/issues).
+
+### Testgrid Integration
+
+Huawei cloud provider is reporting conformance test results to TestGrid as per the [Reporting Conformance Test Results to Testgrid KEP](https://github.com/kubernetes/enhancements/blob/6427a0becff459815e0e41f72f65ab5f3b8e9c6d/keps/sig-cloud-provider/0018-testgrid-conformance-e2e.md). 
```

Comment:
can you provide link to the testgrid results/dashboard?

Reply:
Here is the [testgrid dashboard](https://testgrid.k8s.io/conformance-cloud-provider-huaweicloud).

I'll update to the KEP later.

Reply:
Done.